src/checkers/ast.rs

```diff
@@ -1231,6 +1231,28 @@ where
                     }
 
                     if let Some(asname) = &alias.node.asname {
```

Comment:
In case you haven't caught it: this block only runs when the `as` is set (e.g. `from foo import bar as baz`, but not `from foo import bar`). I think you can lift all these changes out of this `if`, maybe?

Reply:
ohhhhhhh, yeah, I see it now. I put it under that condition because I thought the original call was under the same clause on line 896, but I miss-read it. It's not under the `if let Some(asname) = &alias.node.asname` so mine shouldn't be either. Makes sense.

Reply:
(Just for clarity in communication: do you plan to fix this, or want me to?)